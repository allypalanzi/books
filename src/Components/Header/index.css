.c-header {
  text-align: center;
  background-color: #046B99;
  color: #fff;
  letter-spacing: 1px;
}