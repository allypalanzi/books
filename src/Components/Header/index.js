import React from 'react';
import { Link } from 'react-router-dom';
import './index.css';

/*
* A header component renders the application header
*/

const Header = props => {
  return (
    <header className="c-header">
      <Link to="/"><h1>Science Books <span role="img" aria-label="Science emojis, scientist, microscope, books">👩‍🔬🔬📚</span></h1></Link>
    </header>
  );
}

export default Header;
