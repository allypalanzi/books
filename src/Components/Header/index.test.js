import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import Header from './';
// MemoryRouter is needed for Link to work
import { MemoryRouter } from 'react-router-dom';

describe('<Header/ >', () => {
  const component = (
    <MemoryRouter>
      <Header />
    </MemoryRouter>
  );
  const renderedComponent = renderer.create(component);

  it('renders correctly', () => {
    expect(renderedComponent).toMatchSnapshot();
  });
});
