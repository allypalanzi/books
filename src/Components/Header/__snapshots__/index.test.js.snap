// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header/ > renders correctly 1`] = `
<header
  className="c-header"
>
  <a
    href="/"
    onClick={[Function]}
  >
    <h1>
      Science Books 
      <span
        aria-label="Science emojis, scientist, microscope, books"
        role="img"
      >
        👩‍🔬🔬📚
      </span>
    </h1>
  </a>
</header>
`;
